### ISSUE TITLE
How can prune before training?

### ISSUE THREAD

**Issue by ghost** [kentaroy47/Deep-Compression.Pytorch#4]
How to prune the existing model before training? That means I don't need to training to keep weight zero.
I can get good accuracy on testloader after pruning. Now, I am confused no matter which percentage, test on testloader only 10% accuracy. Where is the wrong?

**Comment by kentaroy47** [comment]
Can you elaborate on the issue?

**Comment by ghost** [comment]
I solve this problem. In this code, that `weights=abs(torchweights)` and return after abs weight. If not prune the existing model before training, use ```weights=torchweights.cpu().numpy()  weights_abs=np.abs(torchweights.cpu().numpy())``` instead of the first two code.
